src/components/Datepicker.vue

```diff
@@ -89,45 +89,27 @@ export default {
         return val === null || val instanceof Date || typeof val === 'string'
       }
     },
-    name: {
-      value: String
-    },
-    id: {
-      value: String
-    },
+    name: String,
```

Comment:
Has this always been a thing in Vue? I can't find any documentation on it.
Neat though.